### PR TITLE
Tighten contributing instructions.

### PR DESCRIPTION
**Pull request by monopole**
Attempt to clarify request/proposal/impl cycle and @mention usage.
Cut ~700 words (~30%), fixed some links.

### CODE REVIEW

**Comment by pwittrock** [comment]
cc @pwittrock

File: sig-cli/contributing.md [hunk 1]
```diff
@@ -1,328 +1,297 @@
...
+   > contributing to the Kubernetes project.  I have read all of the
+   > user documentation listed on the community contributring page.
+   > What should I do next to get started?
+- Attend a sig-cli [meeting] and introduce yourself as looking to get started. 
+  
+### Bug lifecycle
+
+1. An [issue] is filed that
+  - includes steps to reproduce the issue including client / server version,
+  - mentions `@kubernetes/sig-cli-bug`.
```
Comment (pwittrock) [inline sig-cli/contributing.md]: bugs

File: sig-cli/contributing.md [hunk 2]
```diff
@@ -1,328 +1,297 @@
...
+2. A [PR] fixing the issue is implemented that
+  - __includes unit and e2e tests__,
+  - incorporates review feedback,
+  - description includes `Closes #<Issue Number>`,
+  - description or comment @mentions `@kubernetes/sig-cli-pr-review`.
```
Comment (pwittrock) [inline sig-cli/contributing.md]: reviews

File: sig-cli/contributing.md [hunk 3]
```diff
@@ -1,328 +1,297 @@
...
-  - Describe steps to reproduce the issue including client / server version
-2. Implement the fix
-  - Send a PR
-  - Add `Closes #<Issue Number>` to the description
-  - Mention `@kubernetes/sig-cli-pr-reviews` in a comment
-  - Incorporate review feedback
-  - **Note:** Include unit and e2e tests
-7. Release
-  - Wait for your feature to appear in the next Kubernetes release!
+A feature request is an [issue] mentioning `@kubernetes/sig-cli-feature-request`.
```
Comment (pwittrock) [inline sig-cli/contributing.md]: requests

File: sig-cli/contributing.md [hunk 4]
```diff
@@ -1,328 +1,297 @@
...
-  been disabled and pushed into the next release.
-- support level for items has been properly defined - alpha / beta / ga
-- api changes have been properly vetted by the appropriate parties
-- user facing documentation has been updated
-
-**Note:** in most cases this will be done by sig leads on behalf of the
-other community members.
-
-## Implementing the feature
+1. Identify a problem by filing an [issue] (mention `@kubernetes/sig-cli-feature-request`).
```
Comment (pwittrock) [inline sig-cli/contributing.md]: requests

File: sig-cli/contributing.md [hunk 5]
```diff
@@ -1,328 +1,297 @@
...
+
+A design proposal is a single markdown document in the [design repo]
+that follows the [design template].
+
+To make one,
+- Prepare the markdown document as a PR to that repo.
+  - Avoid _Work In Progress_ (WIP) PRs (send it only after
+    you consider it complete).
+  - For early feedback, use the email discussion [group].
+- Mention `@kubernetes/sig-cli-proposal` in the description.
```
Comment (pwittrock) [inline sig-cli/contributing.md]: proposals

File: sig-cli/contributing.md [hunk 6]
```diff
@@ -1,328 +1,297 @@
...
+the [design proposal] has been merged_.
+
+See the [development guide] for instructions on setting up the
+Kubernetes development environment.
 
-Go [here](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md)
-for instructions on setting up the Kubernetes development environment.
+Implementation PRs should
+- mention the issue of the associated design proposal,
+- mention `@kubernetes/sig-cli-pr-review`,
```
Comment (pwittrock) [inline sig-cli/contributing.md]: reviews

File: sig-cli/contributing.md [hunk 7]
```diff
@@ -1,328 +1,297 @@
...
-escalate by contacting sig leads and attending the bi-weekly sig-cli
-meeting.
+escalate by contacting sig [leads] and attending the [meeting].
 
-## What to do if your feature request issue is stuck
+### If your feature request issue is stuck
 
 If an issue isn't getting any attention and is unresolved, mention
-@kubernetes/sig-cli-feature-requests.
+`@kubernetes/sig-cli-feature-request`.
```
Comment (pwittrock) [inline sig-cli/contributing.md]: s

File: sig-cli/contributing.md [hunk 8]
```diff
@@ -1,328 +1,297 @@
...
+  - Not something we plan to implement in the proposed manner.
+  - Not something we plan to revisit in the next 12 months.
+
+## Escalation
+
+### If your bug issue is stuck
 
 If an issue isn't getting any attention and is unresolved, mention
-@kubernetes/sig-cli-bugs.
+`@kubernetes/sig-cli-bug`.
```
Comment (pwittrock) [inline sig-cli/contributing.md]: s

File: sig-cli/contributing.md [hunk 9]
```diff
@@ -1,328 +1,297 @@
...
+See the sig-cli [community page] for points of contact and meeting times:
+
+- attend the sig-cli [meeting]
+- message one of the sig leads on [slack]
+- send an email to the _kubernetes-sig-cli@googlegroups.com_ [group].
+
+## Use of @mentions
+
+- `@{any lead}` solicit opinion or advice from [leads].
+- `@kubernetes/sig-cli-bug` sig-cli centric bugs.
```
Comment (pwittrock) [inline sig-cli/contributing.md]: plural

File: sig-cli/contributing.md [hunk 10]
```diff
@@ -1,328 +1,297 @@
...
 ## TL;DR
 
-- New contributors should reach out on slack sig-cli - [link](https://github.com/kubernetes/community/tree/master/sig-cli) for an issue to work on that has an approved design
-- New contributors should attend the next sig-cli meeting and introduce themselves
-- Write tests
+- The sig-cli [community page] lists sig-cli [leads],
+  channels of [communication], and group [meeting] times.
+- New contributors: please start by adopting an [existing issue].
+- Request a feature by making an [issue] and mentioning
+  `@kubernetes/sig-cli-feature-request`.
```
Comment (pwittrock) [inline sig-cli/contributing.md]: requests

File: sig-cli/contributing.md [hunk 11]
```diff
@@ -1,328 +1,297 @@
...
+proposal_ is much less likely to be stuck than a dangling PR.
+
+However, if it happens do the following:
 
 - If your PR is stuck for a week or more because it has never gotten any
-  comments, mention @kubernetes/sig-cli-pr-reviews and ask for attention.
-  If this is not successful in a day or two, escalate to sig leads or
-  attend the bi-weekly meeting.
-- If your PR is stuck for a week or more after it got comments, but
+  comments, mention `@kubernetes/sig-cli-pr-review` and ask for attention.
```
Comment (pwittrock) [inline sig-cli/contributing.md]: s

**Comment by monopole** [comment]
Abandoning this in favor of https://github.com/kubernetes/community/pull/383
